workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/EntityGameObjectCreator.cs

```diff
@@ -27,7 +26,8 @@ public EntityGameObjectCreator(World world, Dictionary<string, GameObject> initi
                 prefab = Resources.Load<GameObject>(prefabPath);
                 if (prefab == null)
                 {
-                    throw new PrefabNotFoundException($"Prefab for prefabPath {prefabPath} not found.");
+                    return null;
```

Comment:
What is the reasoning behind this change? We are not even checking for null in the return value in `GameObjectInitializationSystem`.

Reply:
I agree. This should remain an exception, especially if it's trying to load a prefab from a path that does not contain a prefab.

Reply:
Also commented code 😬

Reply:
ok, in that case I'll also implement that check inside the interface here, because there are definitely entities where we don't want to create a prefab